unreleased/remove_jaeger_remote_sampling_from_receiver_#6633.yaml

```diff
@@ -0,0 +1,11 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
```

Comment:
This is a breaking change, following up on a previous deprecation.

Reply:
fixed :+1: